docs/commands.md

```diff
@@ -31,47 +31,80 @@ run_cmd tactic.skip -- this serves as a "barrier" between `import` and `#find`
 
 ## find
 
-The `find` command from `tactic.find` allows to find definitions lemmas using
+[[source]](../src/tactic/find.lean) [mathlib user command: `import tactic.find` or `import tactic.basic`]
```

Comment:
Why this is called “mathlib user command”? I'd expect something like “import with:”.

Reply:
Good point. I can see how this is confusing. Also, since all the commands documented in `commands.md` are user commands provided by mathlib, the "mathlib user command" label is indeed redundant. Would the following be better formats?

At the start of each user command doc:

[source] [import with `import tactic.find` or `import tactic.basic`]

At the start of each tactic doc:

[source] [Lean core library tactic, auto-imported by default]

or

[source] [mathlib basic tactic, import with `import tactic.core` or `import tactic.basic`]

or

[source] [mathlib tactic, import with `import tactic.norm_num` or `import tactic`]

Reply:
I agree that "mathlib user command" is misleading. The suggestions looks fine.